.github/workflows/ci.yml

```diff
@@ -46,28 +67,17 @@ jobs:
         run: |
           python -m pip install --upgrade pipenv
           pipenv install --dev
-          mkdir "${GITHUB_WORKSPACE}/bin"
-          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b "${GITHUB_WORKSPACE}/bin"
-          chmod +x "${GITHUB_WORKSPACE}/bin/syft"
-          echo "${GITHUB_WORKSPACE}/bin" >> "${GITHUB_PATH}"
-      - name: Install cosign
-        uses: sigstore/cosign-installer@main
       - name: Build the images
-        run: pipenv run invoke build --debug
-      - name: Generate SBOMs
-        run: pipenv run invoke sbom --debug
-      - uses: actions/upload-artifact@v2
-        with:
-          name: SBOMs
-          path: sbom.*.spdx.json
-          if-no-files-found: error
+        run: pipenv run invoke build
       - name: Log in to Docker Hub
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Publish the latest images to Docker Hub
-        run: pipenv run invoke publish latest --debug
+        run: pipenv run invoke publish latest
+      - name: Install cosign
```

Comment:
This moves cosign installation to happen after building, which seems more appropriate since it will never be needed if the build fails.

Reply:
Agreed.